pre_commit_hooks/file_contents_sorter.py

```diff
@@ -37,13 +40,20 @@ def sort_file_contents(f: IO[bytes]) -> int:
 def main(argv: Optional[Sequence[str]] = None) -> int:
     parser = argparse.ArgumentParser()
     parser.add_argument('filenames', nargs='+', help='Files to sort')
+    parser.add_argument(
+        '--ignore-case',
+        action='store_const',
+        const=bytes.lower,
+        default=None,
+        help='fold lower case to upper case characters',
```

Comment:
(Argument name and help happen to be grabbed from sort(1) from coreutils, though I didn't also add `-f`, since that's just confusing... can though if desired.)